docs/source/hardware/lowpriority.rst

```diff
@@ -0,0 +1,79 @@
+
+.. _partition_names:
+
+**********
+Low priority partition
+**********
+:sub:`Updated September, 2022`
+
+
+Introduction
+===================
+``lowpriority`` is a new partition on Discovery that allows the research community to use private resources on the HPC cluster when they are idle. 
```

Comment:
@ShobanaSekar, it might be better to not use "Discovery" and replace that with HPC cluster. So something like:

``lowpriority`` is a new partition on our high performance computing (HPC) cluster that allows the research community to use private resources when they are idle.

Reply:
@manasvita I've partly addressed this but still think its okay to leave "Discovery" in there, because users might be able to relate to "Discovery" more that "high performance computing". When we get to the stage of migrating things to the next cluster  we can just substitute "Discovery" with the new cluster's name at that point? Let me know if you have strong opinions on this and we can figure it out, thanks!

Reply:
@ShobanaSekar, sure, that sounds good to me.